src/config/constants/farms.ts

```diff
@@ -43,27 +43,29 @@ const farms: SerializedFarmConfig[] = [
   },
   //    * V3 by order of release (some may be out of PID order due to multiplier boost)
   {
-    pid: 40,
-    v1pid: 390,
-    lpSymbol: 'CHR-BUSD LP',
+    pid: 102,
+    lpSymbol: 'PEAK-BNB LP',
     lpAddresses: {
       97: '',
-      56: '0x6045931e511ef7e53a4a817f971e0ca28c758809',
+      56: '0x41140a1650372Fb8cb2f71e335448ab8cfc1c4f3',
     },
-    token: serializedTokens.chr,
-    quoteToken: serializedTokens.busd,
+    token: serializedTokens.peak,
+    quoteToken: serializedTokens.wbnb,
+    isCommunity: true,
+    auctionHostingStartSeconds: 1654772400,
   },
   {
-    pid: 38,
-    v1pid: 386,
-    lpSymbol: 'HOTCROSS-BNB LP',
+    pid: 93,
+    v1pid: 524,
+    lpSymbol: 'HAPPY-BNB',
     lpAddresses: {
       97: '',
-      56: '0xf23bad605e94de0e3b60c9718a43a94a5af43915',
+      56: '0x008604A38cD589680F7B8f085DC2D5B4F81151dB',
     },
-    token: serializedTokens.hotcross,
+    token: serializedTokens.happy,
     quoteToken: serializedTokens.wbnb,
-    auctionHostingStartSeconds: 1653649200,
+    isCommunity: true,
+    auctionHostingStartSeconds: 1654772400,
```

Comment:
No need for this property if it is isCommunity already

Reply:
@memoyil updated.